goto/gotomagic/magic.py

```diff
@@ -92,7 +92,7 @@ def get_uri(self, magicword):
                 text.warning.messages["magicword_does_not_exist"],
                 magicword=magicword
             )
-            return None
+            exit(1)
```

Comment:
Speaking of separation of concerns. I think it is fair to say that the Database layer should not `print text` and exit the program, but rather return errors and let the application layer decide whats printed and when to exit.

I am just mentioning it, fully aware that it might be outside the scope of this PR ;)

Reply:
I have addressed the very same concern in earlier Pull Requests, and as this comment left in the codebase suggests: 
https://github.com/technocake/goto/blob/master/goto/commands/default.py#L13-L14

I totally agree! And it would require a bit of refactoring

Reply:
I think we should rely on raising exceptions over returning errors. 
If they are uncaught, they will ultimately exit the program. That is fail fast. Good.